src/nrc/datamodel/management/commands/clean_old_notifications.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from ...models import Notificatie
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        date_limit = timezone.now() - timedelta(
+            days=settings.NOTIFICATION_NUMBER_OF_DAYS_RETAINED
+        )
+
+        notifications_filtered = Notificatie.objects.filter(
+            forwarded_msg__aanmaakdatum__lt=date_limit
+        ).delete()
+
+        self.stdout.write(
+            str(notifications_filtered[0])
+            + " notifications have been deleted : "
+            + str(notifications_filtered[1])
+        )
```

Comment:
```suggestion
        self.stdout.write(f"{notifications_filtered[0]} notifications have been deleted")
```

I think the number of deleted notifications is enough, the dictionary is no needed to display

Reply:
This is the result with the dictionary :
![image](https://github.com/open-zaak/open-notificaties/assets/35032566/19fc528f-21c2-4878-87c8-eb43104a4b55)
It seems relevant to me, to know for NotificatieResponse